src/js/components/Banner/ShabadavaliBanner.tsx

```diff
@@ -2,36 +2,26 @@ import React from 'react';
 import Banner from './Banner';
 
 const ShabadavaliBanner = () => {
-
-  const {
-    isFetchingData: isFetchingBannerMessage,
-    data: bannerData,
-  } = {
-    isFetchingData: false,
-    data: {
-      active: true,
-      data: {
-        message: "Help us preserve the Punjabi language! Donate on Dasvandh Network and double your impact with matching!",
-        link: "dvnetwork.org/learnpunjabi",
-        label: "Donate now",
-        type: "3",
-      }
-    }
+  const bannerData = {
+    message: "Help us preserve the Punjabi language! Donate on Dasvandh Network and double your impact with matching!",
+    link: "dvnetwork.org/learnpunjabi",
+    label: "Donate now",
+    type: "3",
   };
 
-  const lastSeen = sessionStorage.getItem("bannerMessage");
+  const lastSeen = sessionStorage.getItem("shbdvliBannerMsg");
```

Comment:
@MySmilingTurban 😅 equally good with adding the complete name `shabdavliBannerMessage`. More clear as we already bannerMessage state.